lib/improver/tests/utilities/test_temporal.py

```diff
@@ -241,13 +241,6 @@ class Test_datetime_constraint(Test_common_functions):
     object.
     """
 
-    def test_constraint_equality(self):
```

Comment:
I don't think there is any way of doing this now that our constraint is a function. I'm not sure how we check they are equal.